pages/portfolio.html

```diff
@@ -4,9 +4,65 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Welcome to my Webpage</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <div id="grid"> 
+        <div id="Home"> <a href="index.html">Home</a> </div>
+        <div id="About"> <a href="about.html">About</a> </div> 
+        <div id="Portfolio"> <a href="portfolio.html">Portfolio</a> </div>
+      </div>
+    </nav>  
+  </header>
+  <main>
+    <article>
+      <section>
+        <h1>Task List API Project</h1> 
+        <p>
+          <img alt="A note book with Task List written at the top of the page." src="../assets/task-list-template.jpg" width="600" height="350" class="center">
+        </p>
+        <p>Task List is a project I completed where users can organize and track our goals into smaller tasks in order to complete them. Users are able to sort tasks, mark tasks as complete, get feedback about our task list through Slack, organize tasks with goals, and so much more. There are options to create, read, update, and delete tasks in order to help stay on track with goals.</p>
+        <a href="https://github.com/sabraha0311/task-list-api">Click here to visit my Task List API project</a>
+      </section>
+
+      <section>
+        <h1>Swap Meet Project</h1>
```

Comment:
Generally there should just be one `h1` tag on a page since it denotes the biggest header.